Index: Hotteling.API/Program.cs
===================================================================
diff --git a/Hotteling.API/Program.cs b/HotellingSystem_TIN_s31722/Program.cs
rename from Hotteling.API/Program.cs
rename to HotellingSystem_TIN_s31722/Program.cs
--- a/Hotteling.API/Program.cs	(revision 947f824ea7a9258bdc8487a0465b0f55d6e78aa8)
+++ b/HotellingSystem_TIN_s31722/Program.cs	(date 1766669643278)
@@ -1,4 +1,4 @@
-namespace Hotteling.API;
+namespace HotellingSystem_TIN_s31722;
 
 public class Program
 {
@@ -26,6 +26,11 @@
 
         app.UseAuthorization();
 
+        var summaries = new[]
+        {
+            "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+        };
+        
         app.Run();
     }
 }
\ No newline at end of file
diff --git a/Hotteling.API/Properties/launchSettings.json b/Hoteling.API/Properties/launchSettings.json
rename from Hotteling.API/Properties/launchSettings.json
rename to Hoteling.API/Properties/launchSettings.json
